gdk4-wayland/README.md

```diff
@@ -10,9 +10,8 @@ Currently, the minimum supported Rust version is `1.48.0`.
 
 ## Documentation
 
-- [Rust API - Stable] TODO
-- [Rust API - Development](https://gtk-rs.org/gtk4-rs/git/docs/gdk4_wayland/)
-- [C API](https://docs.gtk.org/gdk4-wayland/)
+- Rust API [Stable]/[Development](https://gtk-rs.org/gtk4-rs/docs/git/gdk4_wayland/)
+- [The C API](https://docs.gtk.org/gdk4-wayland/)
```

Comment:
Some are using `The C API` and others like the gtk4 readme are using `C API` it  would be nice to keep them consistent if possible

Reply:
Mmmh sure.
Which one do you prefer?

Reply:
The C API is fine i think

Reply:
Changed